tools/StaticAnalysis/Exceptions/AzureRM.Sql/SignatureIssues.csv

```diff
@@ -189,3 +189,15 @@
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Sql\Microsoft.Azure.Commands.Sql.dll","Microsoft.Azure.Commands.Sql.Advisor.Cmdlet.GetAzureSqlDatabaseAdvisor","Get-AzureRmSqlDatabaseAdvisor","1","8600","Cmdlet 'Get-AzureRmSqlDatabaseAdvisor' has no defined output type.","Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Sql\Microsoft.Azure.Commands.Sql.dll","Microsoft.Azure.Commands.Sql.Advisor.Cmdlet.GetAzureSqlDatabaseAdvisor","Get-AzureRmSqlDatabaseAdvisor","1","8410","Parameter ExpandRecommendedActions of cmdlet Get-AzureRmSqlDatabaseAdvisor does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Sql\Microsoft.Azure.Commands.Sql.dll","Microsoft.Azure.Commands.Sql.Advisor.Cmdlet.SetAzureSqlDatabaseAdvisorAutoExecuteStatus","Set-AzureRmSqlDatabaseAdvisorAutoExecuteStatus","1","8600","Cmdlet 'Set-AzureRmSqlDatabaseAdvisorAutoExecuteStatus' has no defined output type.","Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Sql\Microsoft.Azure.Commands.Sql.dll","Microsoft.Azure.Commands.Sql.VulnerabilityAssessmentSettings.Cmdlet.GetAzureSqlDatabaseVulnerabilityAssessmentSettings","Get-AzureRmSqlDatabaseVulnerabilityAssessmentSettings","1","8400",""Get-AzureRmSqlDatabaseVulnerabilityAssessmentSettings uses the noun 'AzureRmSqlDatabaseVulnerabilityAssessmentSettings'"," which does not follow the enforced naming convention of using a singular noun for a cmdlet name."","Consider using a singular noun for the cmdlet name."
```

Comment:
You currently have:
```
"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Sql\Microsoft.Azure.Commands.Sql.dll","Microsoft.Azure.Commands.Sql.VulnerabilityAssessmentSettings.Cmdlet.GetAzureSqlDatabaseVulnerabilityAssessmentSettings","Get-AzureRmSqlDatabaseVulnerabilityAssessmentSettings","1","8400",""Get-AzureRmSqlDatabaseVulnerabilityAssessmentSettings uses the noun 'AzureRmSqlDatabaseVulnerabilityAssessmentSettings'"," which does not follow the enforced naming convention of using a singular noun for a cmdlet name."","Consider using a singular noun for the cmdlet name."
```

It needs to be:
```
"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Sql\Microsoft.Azure.Commands.Sql.dll","Microsoft.Azure.Commands.Sql.VulnerabilityAssessmentSettings.Cmdlet.GetAzureSqlDatabaseVulnerabilityAssessmentSettings","Get-AzureRmSqlDatabaseVulnerabilityAssessmentSettings","1","8400","Get-AzureRmSqlDatabaseVulnerabilityAssessmentSettings uses the noun 'AzureRmSqlDatabaseVulnerabilityAssessmentSettings' which does not follow the enforced naming convention of using a singular noun for a cmdlet name.","Consider using a singular noun for the cmdlet name."
```

Do this fix for all the lines that you added.